### PR TITLE
SDIT-2173 Migrate contact email, employments and identifiers

### PR DESCRIPTION
**Pull request by andymarke**

### CODE REVIEW

File: src/main/kotlin/uk/gov/justice/digital/hmpps/prisonerfromnomismigration/contactperson/ContactPersonMigrationService.kt [hunk 1]
```diff
@@ -119,15 +123,17 @@ class ContactPersonMigrationService(
   }
 }
 
+@Suppress("UNNECESSARY_NOT_NULL_ASSERTION")
```
Comment (andymarke) [inline src/main/kotlin/uk/gov/justice/digital/hmpps/prisonerfromnomismigration/contactperson/ContactPersonMigrationService.kt:126]: stupid IntelliJ Kotlin compile bug - doing a clan each time was slowing me down no end

File: src/test/kotlin/uk/gov/justice/digital/hmpps/prisonerfromnomismigration/contactperson/ContactPersonMigrationIntTest.kt [hunk 2]
```diff
@@ -500,6 +557,62 @@ class ContactPersonMigrationIntTest : SqsIntegrationTestBase() {
...
+        assertThat(person.employments).hasSize(1)
+        with(person.employments!![0]) {
+          assertThat(sequence).isEqualTo(1)
+          assertThat(corporate.id).isEqualTo(120)
+          assertThat(corporate.name).isEqualTo("Police")
+          assertThat(active).isTrue()
+          assertThat(createUsername).isEqualTo("ADJUA.BEEK")
+          assertThat(createDateTime).isEqualTo(LocalDateTime.parse("2022-02-02T10:23"))
+          assertThat(modifyUsername).isEqualTo("ADJUA.MENSAH")
+          assertThat(modifyDateTime).isEqualTo(LocalDateTime.parse("2024-02-02T10:23"))
```
Comment (karenmillermoj) [inline src/test/kotlin/uk/gov/justice/digital/hmpps/prisonerfromnomismigration/contactperson/ContactPersonMigrationIntTest.kt:573]: I thought we weren't sending audit data anymore?

**Review by karenmillermoj — APPROVED** [review]
lgtm

File: src/test/kotlin/uk/gov/justice/digital/hmpps/prisonerfromnomismigration/contactperson/ContactPersonMigrationIntTest.kt (hunk 2 above)
Comment (andymarke) [reply src/test/kotlin/uk/gov/justice/digital/hmpps/prisonerfromnomismigration/contactperson/ContactPersonMigrationIntTest.kt:573]: Contacts haven't changed this yet - unlike CSIP

**Pull request merged**